### DISCUSSION TITLE
Missing Update

### DISCUSSION THREAD

**Discussion by mrargold** [osmandapp/OsmAnd#14652] [Maps Related]
I notice that all of my maps except for "United States Florida" have received updates to June 1 but the FL map remains at May 1. Is there anyone I can nudge/nag about this?

**Comment by vshcherb** [comment]
Indeed florida update didn't happen. Investigating it. I will rerun generation process https://builder.osmand.net:8080/view/Generate%20Maps/job/Maps_GenerateFullMaps/1276/console

**Reply by mrargold** [reply]
The June 1 map of FL has landed! Many thanks.

**Reply by mrargold** [reply]
The map of August 1 for  United States Florida is still awaited. Does this need another nudge this month?

**Reply by vshcherb** [reply]
Will be updated tomorrow and next month should be at 13th

**Reply by mrargold** [reply]
Many thanks.
